packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -548,7 +548,8 @@ export function isRuntimeMessage(message: ISequencedDocumentMessage): boolean {
 }
 
 export function unpackRuntimeMessage(message: ISequencedDocumentMessage) {
-    if (message.metadata?.compressed) {
+    if (message.metadata?.compressed &&
+        message.contents.type !== RuntimeMessage.ChunkedOp) {
```

Comment:
I'd rather move compression out of unpackRuntimeMessage() into its own function, to run after this.processRemoteChunkedMessage() call, then have two places that do decompression.
This would require other places (that use unpackRuntimeMessage) to call that new function, but that probably fine (note that code in fluidAnalyzeMessages.ts will not work either way and likely needs to be fixed, if we want for it to work with compressed chunked ops).